identity-engine/embedded-sign-in-widget/server/server.go

```diff
@@ -214,6 +216,46 @@ func (s *Server) LoginCallbackHandler(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprintln(w, "The state was not as expected")
 		return
 	}
+
+	// Check if interaction_required error is returned
```

Comment:
If we get `interaction_required` error from the backend, it means further remediation steps are required to complete login (say after logging to google IdP). The widget will handle these remediations if we provide the saved interaction_handle.